assets/css/style.css

```diff
@@ -213,8 +211,8 @@ label {
   font-family: "Poppins", sans-serif;
   font-style: normal;
```

Comment:
- [ ] Please make sure your background for the `main-content` section is displayed. kindly check

## your design👇
![deskbackst](https://user-images.githubusercontent.com/66992901/193130714-2e6bda18-692f-4b69-aae7-200972d997a4.PNG)

## Figma design👇
![deskback](https://user-images.githubusercontent.com/66992901/193130789-7435ae57-05b1-4a1b-bd2c-cb2324979869.PNG)